solution-base/deps.txt

```diff
@@ -0,0 +1,9 @@
+kubedb/operator:v0.13.0-rc.0
+kubedb/redis:5.0.3-v1
+kubedb/mongo:4.0-v1
+kubedb/mongo-tools:4.0-v1
+kubedb/busybox:latest
+kubedb/mongodb_exporter:v1.0.0
+kubedb/redis:5.0.3-v1
+redis:5.0.3
```

Comment:
not sure where this is used.
if by the zenko deployed redis, wouldn't it be better to have this in the zenko solution?

Reply:
I pulled it from the zenko operators zenko-version YAML, but it does make sense to put it in the zenko operator iso. Does this mean I would want the MongoDB stuff as well in the zenko ISO? Not really sure where to draw the line I guess is my question.

Reply:
the MongoDB images can stay.  it's only `redis:5.0.3`/`oliver006/redis_exporter:v1.8.0` that will need to be copied to the `Zenko` solution.

Reply:
ok will do ( setting up a metal cluster atm )